systemtests/src/main/java/com/github/streamshub/systemtests/utils/playwright/PwUtils.java

```diff
@@ -0,0 +1,155 @@
+package com.github.streamshub.systemtests.utils.playwright;
+
+import com.github.streamshub.systemtests.Environment;
+import com.github.streamshub.systemtests.TestCaseConfig;
+import com.github.streamshub.systemtests.constants.TimeConstants;
+import com.github.streamshub.systemtests.enums.BrowserTypes;
+import com.github.streamshub.systemtests.exceptions.SetupException;
+import com.github.streamshub.systemtests.logs.LogWrapper;
+import com.github.streamshub.systemtests.utils.playwright.locators.CssSelectors;
+import com.microsoft.playwright.Browser;
+import com.microsoft.playwright.BrowserType;
+import com.microsoft.playwright.Locator;
+import com.microsoft.playwright.Page;
+import com.microsoft.playwright.Playwright;
+import com.microsoft.playwright.options.WaitForSelectorState;
+import com.microsoft.playwright.options.WaitUntilState;
+import io.skodjob.testframe.TestFrameConstants;
+import io.skodjob.testframe.wait.Wait;
+import org.apache.logging.log4j.Logger;
+
+import java.util.Locale;
+import java.util.regex.Pattern;
+
+public class PwUtils {
+    private static final Logger LOGGER = LogWrapper.getLogger(PwUtils.class);
+    private PwUtils() {}
+
+    public static Browser createBrowser(Playwright playwright) {
+        BrowserType.LaunchOptions options = new BrowserType.LaunchOptions().setHeadless(Environment.RUN_HEADLESS);
+        BrowserTypes browserType = BrowserTypes.valueOf(Environment.BROWSER_TYPE.toUpperCase(Locale.ENGLISH));
+
+        return switch (browserType) {
+            case CHROMIUM -> playwright.chromium().launch(options);
+            case MSEDGE -> playwright.chromium().launch(options.setChannel("msedge"));
+            case FIREFOX -> playwright.firefox().launch(options);
+            case WEBKIT -> playwright.webkit().launch(options);
+            default -> throw new SetupException("Cannot create Playwright browser type: " + browserType);
+        };
+    }
+
+    public static void login(TestCaseConfig tcc) {
+        final String loginUrl = PwPageUrls.getKafkaLoginPage(tcc, tcc.kafkaName());
+        LOGGER.info("Logging in to the Console with URL: {}", loginUrl);
+        waitForConsoleUiToBecomeReady(tcc);
+        // Anonymous login
+        tcc.page().navigate(loginUrl, getDefaultNavigateOpts());
+        tcc.page().waitForURL(Pattern.compile(loginUrl + ".*"), getDefaultWaitForUrlOpts());
+        waitForLocatorVisible(tcc, CssSelectors.LOGIN_ANONYMOUSLY_BUTTON);
+        tcc.page().click(CssSelectors.LOGIN_ANONYMOUSLY_BUTTON);
+        // Go to overview page
+        tcc.page().waitForURL(PwPageUrls.getOverviewPage(tcc, tcc.kafkaName()), getDefaultWaitForUrlOpts());
+        LOGGER.info("Successfully logged into Console");
+    }
+
+    public static String getTrimmedText(String text) {
+        return text.replaceAll("\\n", "")
+            .replaceAll("[\\h\\s+\\t]", " ")
```

Comment:
I think the `+` is redundant (Sonar flagged this). Do you want to replace multiple occurrences of `\h`, `\s` and `\t` with a single space?

```suggestion
            .replaceAll("[\\h\\s\\t]", " ")
```

Reply:
I think i should use the `\s{2,}` instead to get rid of multiple whitespaces and use just one instead

Reply:
`"[\\h\\s\\t]{2,}"` should work also if you want to consider multiple occurrences of any of the listed whitespace.

Reply:
I think having -> whitespace more then once in a row reduced to one +  horizontal or tab chars replaced completely will do.